bcgithook/scripts/post-commit.sh

```diff
@@ -119,6 +119,7 @@ debug "START"
 [[ ! -r "$CONFIG_FILE" ]] && LOG_FILE="$ERR_FILE" \
                           && debug "CONFIG FILE $CONFIG_FILE NOT FOUND - ABORTING" \
                           && exit 1
+# shellcheck source=/dev/null
```

Comment:
same as first comment. what are you expecting to happen

Reply:
yep, as per shellcheck documentation.

For example, check bottom of
https://github.com/koalaman/shellcheck/wiki/SC2034